Mobile/starter_project_matador/lib/features/example/presentation/widgets/data_fill.dart

```diff
@@ -0,0 +1,132 @@
+import 'package:flutter/material.dart';
+import 'input_box.dart';
+import 'signup_button.dart';
+import 'login_text_at_the _end.dart';
+
+class DataFillBox extends StatelessWidget {
+  const DataFillBox({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      height: MediaQuery.of(context).size.height,
+      width: double.infinity,
+      decoration: BoxDecoration(
+        color: Colors.white,
+        borderRadius: BorderRadius.only(
+          topLeft:
+              Radius.circular((28 / 812) * MediaQuery.of(context).size.height),
+          topRight:
+              Radius.circular((28 / 812) * MediaQuery.of(context).size.height),
+        ),
+      ),
+      child: Column(
+        crossAxisAlignment: CrossAxisAlignment.start,
+        children: <Widget>[
+          Padding(
+            padding: EdgeInsets.fromLTRB(
+                (41.0 / 375) * MediaQuery.of(context).size.width,
+                (32.0 / 812) * MediaQuery.of(context).size.height,
+                0.0,
+                0.0),
+            child: Text(
+              'WELCOME',
+              style: TextStyle(
```

Comment:
Similar to the above comment. there are also styles defined inside the core/utils/constants directory. define the textstyles there and use them. This apply for all the styles.

Reply:
I have resolved this problem